iscsi/target.go

```diff
@@ -118,6 +118,11 @@ func SetLunThinProvisioning(tid int, lun int) error {
 	return UpdateLun(tid, lun, map[string]string{"thin_provisioning": "1"})
 }
 
+// DisableWriteCache will set param write-cache to false for the LUN
+func DisableWriteCache(tid int, lun int) error {
```

Comment:
NIT: can you add a link to the reference of the mode page parameters, otherwise the below is just magic :)

Reply:
Added. Thank you.